README.md

```diff
@@ -3,3 +3,36 @@
 # Atlas in house perl modules
 
 Repository includes internal perl modules that is used for processing studies in atlas data production pipeline particularly handling MAGE-TABs, baseline and differential analysis, zooma mappings, release data exports. 
+
+# Changing config files for local setup
+
+Execution of certain scripts when using the conda installation might fail because certain configuration files won't be found. Currently the main work-around for this (sorry, yes, this is a work-around) is to replace the AtlasSiteConfig.yaml and ArrayExpressSiteConfig.yaml for files that point to the necessary content in the machine where this runs. For this, follow these steps (which assume that a conda environment with perl-atlas-modules is installed):
+
+1.- Find the path to your conda env that contains perl-atlas-modules:
+
+```
+PERL_ATLAS_ENV=$(conda env list | grep perl-atlas | awk '{ print $2 }')
+echo $PERL_ATLAS_ENV
```

Comment:
This command didn't work. It assumes that the conda env is called "perl-atlas", right? But mine is called "atlas_curation". I had to set up the PERL_ATLAS_ENV variable manually to proceed.